packages/mermaid/src/diagrams/class/classDiagram.spec.ts

```diff
@@ -264,6 +264,93 @@ class C13["With Città foreign language"]
       const str = 'classDiagram\n' + 'note "test"\n';
       parser.parse(str);
     });
+
+    const keywords = [
+      'direction',
+      'classDiagram',
+      'classDiagram-v2',
+      'namespace',
+      '}',
+      '()',
```

Comment:
Does this have special meaning?

Reply:
Yes, these parentheses put the parser in a `callback_args` state. They actually have higher precedence than strings so this test is important to test the functionality of statements like :
`click Class1 function1(arg1, arg2)`

Reply:
They are recognized as separate tokens, so they probably should be tested separately. Right this line covers only one test case - empty list of arguments inside the notes. Could you add also unbalanced brackets as well? So as to check whether it is possible to write a note:
```
note "a) first things first"
```

Reply:
Very fair, should have paid attention to that. Thanks for catching it!